ansible/roles/operator-pipeline/templates/openshift/tasks/reserve-operator-name.yml

```diff
@@ -5,11 +5,37 @@ metadata:
   name: reserve-operator-name
 spec:
   params:
+    - name: association
     - name: operator_name
+    - name: pyxis_url
+      default: https://pyxis.engineering.redhat.com
   steps:
     - name: reserve-operator-name
-      image: registry.access.redhat.com/ubi8-minimal
+      image: quay.io/redhat-isv/operator-pipelines-images:latest
       script: |
         #! /usr/bin/env bash
-        # POST request to Pyxis
-        echo "ok, 200"
+        echo "Attempting to reserve operator name"
+
+        reserve-operator-name \
+          --association "$(params.association)"  \
+          --cert-path /cert/tls.crt \
+          --key-path /cert/tls.key \
+          --operator-package-name "$(params.operator_name)" \
+          --source "certified-operators"  \
+          --verbose
+
+        if [ $? -eq 0 ]; then
+          echo "Operator name reserved successfully"
+        else
+          echo "Failed to reserve operator name. Name may have already been taken."
+          exit 1
+        fi
```

Comment:
I noticed that `reserve-operator-name` already returns a non-0 exit code on error with descriptive messages. So we can probably remove these lines and add `set -e` at the start of this script to exit automatically.